Execute mvn install have mistake

### Which version of ShardingSphere did you use?

master


```
[ERROR] 2021-09-08 11:39:00.090 [main] o.a.s.s.core.job.FinishedCheckJob - scaling job 1 finish check failed!
java.lang.NullPointerException: null
        at org.apache.shardingsphere.scaling.core.util.ScalingTaskUtil.isProgressCompleted(ScalingTaskUtil.java:56)
        at org.apache.shardingsphere.scaling.core.util.ScalingTaskUtil.almostFinished(ScalingTaskUtil.java:50)
        at org.apache.shardingsphere.scaling.core.job.FinishedCheckJob.execute(FinishedCheckJob.java:55)
        at org.apache.shardingsphere.scaling.core.job.FinishedCheckJobTest.assertExecuteActiveJob(FinishedCheckJobTest.java:83)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.mockito.internal.runners.DefaultInternalRunner$1.run(DefaultInternalRunner.java:68)
        at org.mockito.internal.runners.DefaultInternalRunner.run(DefaultInternalRunner.java:74)
        at org.mockito.internal.runners.StrictRunner.run(StrictRunner.java:39)
        at org.mockito.junit.MockitoJUnitRunner.run(MockitoJUnitRunner.java:161)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
```

--- a/shardingsphere-scaling/shardingsphere-scaling-core/src/test/java/org/apache/shardingsphere/scaling/core/job/FinishedCheckJobTest.java
+++ b/shardingsphere-scaling/shardingsphere-scaling-core/src/test/java/org/apache/shardingsphere/scaling/core/job/FinishedCheckJobTest.java
@@ -76,7 +76,11 @@ public final class FinishedCheckJobTest {
     public void assertExecuteActiveJob() {
         JobInfo jobInfo = new JobInfo(1L);
         jobInfo.setActive(true);
-        jobInfo.setJobParameter("handleConfig:\nruleConfig:\n");
+        jobInfo.setJobParameter("handleConfig:\n"
+                + "  concurrency: 2\n"
+                + "  shardingTables:\n"
+                + "  - ds_0.t_order_$->{0..1}\n"
+                + "ruleConfig:\n");
         List<JobInfo> jobInfos = Collections.singletonList(jobInfo);
         when(scalingAPI.list()).thenReturn(jobInfos);
         when(scalingAPI.getProgress(1L)).thenReturn(Collections.emptyMap());

